post_mod_update.sh

```diff
@@ -23,9 +23,11 @@ echo "Logging in.."
 user_agent="Mozilla/5.0 (X11; Linux x86_64; rv:60.0) Gecko/20100101 Firefox/81.0 KSP2ForumGithubAction/0.1"
 index_response=$(curl 'https://forum.kerbalspaceprogram.com/' -c ./cookies)
 csrf_token=$(echo "$index_response" | pup 'input[name="csrfKey"] attr{value}') 
+echo "$index_response" > ./index_response.html
```

Comment:
We should put all this file logging behind a flag, which should be false by default, so it can be activated on necessity

Reply:
Ok, that makes sense. I thought it doesn't matter because you already leaving message.md and message.html in local directory.